File: LinkedList/StudentList.java
public class StudentList
{
 public static void main(String arg[])
 {
   SingleList s = new SingleList(); 
  
   s.insert("Jane Mary");
   s.insert("Allan Poe");
   s.insert("Mary Paul");
   s.insert("Sister");
   
   System.out.println("Full list");
   s.displaylist();
   
   //remove node
   s.remove();
   
   System.out.println("\nNew list");
   s.displaylist();
   
  }

}

File: LinkedList/c_implementation/student_list.c
#include "singly_linked_list.h"

/**
 * main - Entry point of the program
 *
 * Return: 0 on success
 */
int main()
{
	listnode_t *head = NULL;

	insert_node(&head, "Jane Mary");
	insert_node(&head, "Allan Poe");
	insert_node(&head, "Mary Paul");
	insert_node(&head, "Sister");
	display_list(head);
	printf("\nDeleted_node = %s\n", delete_node(&head));
	display_list(head);
	printf("\nDeleted_node = %s\n", delete_node(&head));
	printf("\nDeleted_node = %s\n", delete_node(&head));
	display_list(head);
	printf("\nDeleted_node = %s\n", delete_node(&head));

	return (0);
}
